src/functions.py

```diff
@@ -6,7 +6,21 @@
 num = input("Enter a number: ")
 num = int(num)
 
+def is_even(num):
+    if (num % 2) == 0:
+        return True
+    if (num % 2) == 1:
```

Comment:
you could actually leave the second if statement (or else) as a boolean is binary in nature, so if it doesn't meet the True condition, the function will automatically return a False. Saves some time and code